src/content/api/events.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Events
+description: Events and related endpoints
+nav:
+  path: API/Events
+  order: 1
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Error from '../../components/Error.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import Badge from '../../components/Badge.astro';
+
+An event is an activity triggered by an action. Some events are native to Centrapay, these [native events](/api/native-event-types) requiring no integrator involvement.
+
+---
+
+<Endpoint
+  path="/api/accounts/{accountId}/events"
+  filename="events-create"
+>
+  ## Create Event <Badge type="experimental"/>
+
+  This endpoint allows you to create an [ Event ](/api/events).
+
+  <Properties>
+    <Property name="type" type="string" required>
+     The type of event. For Promotions, this must match the supplied `eventType`.
+    </Property>
+
+    <Property name="idempotencyKey" type="string" required>
+      Unique key to prevent the same event being processed multiple times.
+    </Property>
+
+    <Property name="data" type="string">
+     Freeform object with relevant metadata.
+    </Property>
+
+     <Property name="test" type="boolean">
+     A flag which is present if the Event is for testing.
+    </Property>
+  </Properties>
+
+  <Properties heading="Errors">
+    <Error code="403" message="EVENT_ALREADY_EXISTS">
+      The `idempotencyKey` has already been used.
```

Comment:
Suggestion: `An event with this idempotencyKey has already been created for the account.`